net/pdnsd/files/pdnsd.init

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh /etc/rc.common
```

Comment:
@karlp means to convert this init script to procd. See https://github.com/openwrt/packages/blob/master/net/i2pd/files/i2pd.init for an example.

Reply:
Fixed done